tests/integration/dhcp_test.py

```diff
@@ -492,6 +525,63 @@ def test_ipv6_dhcp_switch_on_to_off(dhcpcli_up):
     assert not _has_ipv6_auto_nameserver()
 
 
+def test_dhcp_on_bridge0(dhcpcli_up):
+    """
+    Test dynamic IPv4 & IPv6 addresses over a Linux bridge interface.
+
+    Several checks have been ecooperated in the test due to the high time cost.
+    The dynamic IP over the bridge includes the follwing checks:
+        - The dynamic settings have been applied.
+        - IPv4 and IPv6 addresses have been provided by the server.
+        - IPv4 addresses are identical to the original ones which existed on
+        the nic (dhcpcli interface). [XFAIL]
+        - IPv6 addresses are identical to the original ones which existed on
+        the nic (dhcpcli interface). [XFAIL]
+    """
+    port_name = dhcpcli_up[Interface.KEY][0][Interface.NAME]
+
+    time.sleep(5)  # wait to get IP addresses
+    origin_port_state = statelib.show_only((port_name,))
+
+    bridge_state = create_bridge_subtree_state()
+    port_state = {
+        'stp-hairpin-mode': False,
+        'stp-path-cost': 100,
+        'stp-priority': 32,
+    }
```

Comment:
I think we don't required to this anymore. The verification only check defined data.

Reply:
I tried it but it did not work. I was not focusing on this so I left it like this.
I will recheck so I'll have a better answer. (maybe the assert here is not including this)

Reply:
Confirmed, we have a bug in the test assertion. I will take care of that later.

Reply:
Fixed by changing the assert function.